tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -656,10 +663,10 @@ def sum_of_reoccurring_values(x):
     :return: the value of this feature
     :return type: float
     """
-    x = pd.Series(x)
-    value_counts = x.value_counts()
-    doubled_values = value_counts[value_counts > 1]
-    return sum(doubled_values.index * doubled_values)
+
+    unique, counts = np.unique(x, return_counts=True)
+    counts[counts < 2] = 0
+    return np.sum(counts * unique)
```

Comment:
like above I would suggest to `np.sum(counts > 1)`

And the product with `unique` results in another feature actually.